ci-operator/step-registry/openshift/e2e/test/qe/regenerate-cluster-cert/openshift-e2e-test-qe-regenerate-cluster-cert-commands.sh

```diff
@@ -32,6 +33,8 @@ function extract_oc(){
     return 0
 }
 
+sleep 10800
```

Comment:
@wangke19 why sleeping 10800 seconds (3 hours) at the beginning?

Reply:
Will run the cert rotation manually for debugging, the test chain will hang up 3 hours, we can do something with the cluster in Prow CI.